dist/google-music.js

```diff
@@ -279,6 +293,16 @@ proto.rating = {
     if (el && !this.rating._isElSelected(el)) {
       el.click();
     }
+  },
+
+  // Reset the rating
```

Comment:
Statistically if I do the wrong thing 140 times I'll get it right once :+1: :smile_cat: